recipe/session/sessmodels/models.go

```diff
@@ -178,27 +208,25 @@ type NormalisedErrorHandlers struct {
-	UpdateSessionData        func(newSessionData map[string]interface{}) error
-	GetUserID                func() string
-	GetAccessTokenPayload    func() map[string]interface{}
-	GetHandle                func() string
-	GetAccessToken           func() string
-	UpdateAccessTokenPayload func(newAccessTokenPayload map[string]interface{}) error // Deprecated: use MergeIntoAccessTokenPayload instead
-	GetTimeCreated           func() (uint64, error)
-	GetExpiry                func() (uint64, error)
-
-	RevokeSessionWithContext            func(userContext supertokens.UserContext) error
-	GetSessionDataWithContext           func(userContext supertokens.UserContext) (map[string]interface{}, error)
-	UpdateSessionDataWithContext        func(newSessionData map[string]interface{}, userContext supertokens.UserContext) error
-	GetUserIDWithContext                func(userContext supertokens.UserContext) string
-	GetAccessTokenPayloadWithContext    func(userContext supertokens.UserContext) map[string]interface{}
-	GetHandleWithContext                func(userContext supertokens.UserContext) string
-	GetAccessTokenWithContext           func(userContext supertokens.UserContext) string
-	UpdateAccessTokenPayloadWithContext func(newAccessTokenPayload map[string]interface{}, userContext supertokens.UserContext) error // Deprecated: use MergeIntoAccessTokenPayloadWithContext instead
-	GetTimeCreatedWithContext           func(userContext supertokens.UserContext) (uint64, error)
-	GetExpiryWithContext                func(userContext supertokens.UserContext) (uint64, error)
+	RevokeSession               func() error
+	GetSessionDataInDatabase    func() (map[string]interface{}, error)
+	UpdateSessionDataInDatabase func(newSessionData map[string]interface{}) error
+	GetUserID                   func() string
+	GetAccessTokenPayload       func() map[string]interface{}
+	GetHandle                   func() string
+	GetAccessToken              func() string
+	GetTimeCreated              func() (uint64, error)
+	GetExpiry                   func() (uint64, error)
+
+	RevokeSessionWithContext               func(userContext supertokens.UserContext) error
+	GetSessionDataInDatabaseWithContext    func(userContext supertokens.UserContext) (map[string]interface{}, error)
+	UpdateSessionDataInDatabaseWithContext func(newSessionData map[string]interface{}, userContext supertokens.UserContext) error
+	GetUserIDWithContext                   func(userContext supertokens.UserContext) string
+	GetAccessTokenPayloadWithContext       func(userContext supertokens.UserContext) map[string]interface{}
+	GetHandleWithContext                   func(userContext supertokens.UserContext) string
+	GetAccessTokenWithContext              func(userContext supertokens.UserContext) string
+	GetTimeCreatedWithContext              func(userContext supertokens.UserContext) (uint64, error)
+	GetExpiryWithContext                   func(userContext supertokens.UserContext) (uint64, error)
 
```

Comment:
AccessAndFrontendTokenUpdates should be renamed to AccessAndFrontendTokenUpdated

Reply:
Fixed in https://github.com/supertokens/supertokens-golang/pull/273/commits/4a6a9339ba9b17e9383ed7b2704c1a38f36b3b3b